### ISSUE TITLE
Migration and Mixing with type providers

### ISSUE THREAD

**Issue by davidtme** [cmeeren/Facil#7]
I’m having a problem migration away from the SqlClient type provider to Facil - Our project is so big the switch over to Facil can’t be done in one hit while also keeping up with normal development. 

We have mutable databases feeding into mutable projects then mutable apps so changing one database access layer has a knock-on effect.

I was hoping to mix Facil with the SqlClient but as we have to compile our project with FSharp.Compiler.Tools to get around the Type Provider problems it means the anonymous records Facil creates break the build.

```
error FS0010: Unexpected symbol '|' in expression
error FS0604: Unmatched '{'
```

I know FSharp.Compiler.Tools is a dead project and everyone should switch away from it but in a production this is easier said than done.

I see we can use Result type to tell Facil not to create the anonymous records but adding all the types for all the queries is a bit too much.

I’m happy to do the work and add an option to the Result type which creates a new create record type but before I do I just wanted to make sure you would be happy with this as I know at this moment in time (the early stages of development) you would rather have complete control of code and any changes especially big ones.

At the moment our migration is dead in the water because we had to branch our whole project which has quickly become behind with ongoing development. This is in no way Facil fault it's a great tool just we can't use it because of our setup. As the type providers are widely used this addition could help others projects move to Facil.

**Comment by cmeeren** [comment]
This should be a fairly simple change. Thanks for the PR offer, but I think it's probably quicker for me to implement it myself than to review your PR (and that's not even including the time you must spend).

But let me first suggest a workaround: Would it be possible (during migration) to use a separate F# project for the stuff generated by Facil, and compile that using something other than FSharp.Compiler.Tools?

**Comment by davidtme** [comment]
@cmeeren thank you so much 🥇 

I did a try separate project but FSharp.Compiler.Tools throws the following errors:

```
FSC : error FS0229: Error opening binary file 'x.dll': Error reading/writing metadata for the F# compiled DLL 'x.dll'. Was the DLL compiled with an earlier version of the F# compiler? (error: 'lookup_uniq in table istrings out of range, n = 99, sizeof(tab) = 84').
FSC : error FS3160: Problem reading assembly 'x.dll': The exception has been reported. This internal exception should now be caught at an error recovery point on the stack. Original message: Error opening binary file 'x.dll': Error reading/writing metadata for the F# compiled DLL 'x.dll'. Was the DLL compiled with an earlier version of the F# compiler? (error: 'lookup_uniq in table istrings out of range, n = 99, sizeof(tab) = 84').)
```

Hopeful once all the migration is complete we can move away from all this legacy stuff and onto just net5 (or net6, 7 etc)

There could be other fun to be had down the road but one problem at a time.

**Comment by cmeeren** [comment]
Released in v0.2.5. You can now set `result` to `"nominal"`.
